modules/apps/transfer/keeper/forwarding.go

```diff
@@ -16,11 +16,12 @@ import (
 func (k Keeper) forwardPacket(ctx sdk.Context, data types.FungibleTokenPacketDataV2, packet channeltypes.Packet, receivedCoins sdk.Coins) error {
 	var memo string
 
+	// Empty forwarding info and propagate memo to MsgTransfer's memo field or continue on remaining hops.
 	var nextForwardingPath types.Forwarding
 	if len(data.Forwarding.Hops) == 1 {
-		memo = data.Forwarding.Memo
+		memo = data.Forwarding.DestinationMemo
```

Comment:
I think when forwarding, the memo should always be `data.Forwarding.DestinationMemo`? (and then in sendTransfer it is put into the destination memo slot correctly?